mlflow/tracking/client.py

```diff
@@ -17,12 +17,13 @@
 _DEFAULT_USER_ID = "unknown"
 
 
-class MLflowService(object):
+class MLflowClient(object):
     """Client of an MLflow Tracking Server that creates and manages experiments and runs.
```

Comment:
Can we move the `get_service` docstring here?

Reply:
:+1: